gamedata/movedefs.lua

```diff
@@ -502,8 +502,8 @@ local moveDatas = {
 	-- armvang corkarg corthermite
 	HTBOT4 = {
 		crushstrength = 252,
-		footprintx = 4,
-		footprintz = 4,
+		footprintx = 5,
+		footprintz = 5,
```

Comment:
Shouldn't this be called HTBOT5 now? It seemed the name was intended to match the foorptinr size,